lnrpc/rpc.proto

```diff
@@ -1760,6 +1852,8 @@ message FeeReportResponse {
 
     /// The total amount of fee revenue (in satoshis) the switch has collected over the past 1 month.
     uint64 month_fee_sum = 4 [json_name = "month_fee_sum"];
+
```

Comment:
I think we'd want to remove the day/week/month sums from this message, since they're already being provided on a per-chain basis in ChainFeeReport.